src/framework/node.ml

```diff
@@ -1,6 +1,8 @@
 open Cil
 open Pretty
 
+include Printable.Std
```

Comment:
This brings in `let name () = "std"`, which should be overridden here to avoid confusion when using this module inside something else. The previous `PrintableNode` did this correctly.

Other things in `Printable.Std` should be fine by default.